.internal-ci/helm/fog-ingest/values.yaml

```diff
@@ -67,31 +50,56 @@ fogIngest:
     value: 'true'
     effect: NoSchedule
 
-  affinity:
-    podAntiAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-      - weight: 100
-        podAffinityTerm:
-          labelSelector:
-            matchExpressions:
-            - key: app
-              operator: In
-              values:
-              - fog-ingest
-          topologyKey: 'kubernetes.io/hostname'
-
-  initContainers:
-  - name: sysctl
-    image: ubuntu:20.04
-    imagePullPolicy: Always
-    command:
-    - sysctl
-    - -w
-    - net.ipv4.tcp_retries2=5
-    securityContext:
-      privileged: true
-      runAsUser: 0
-      runAsNonRoot: False
+  configMap:
+    data:
+      # https://docs.diesel.rs/diesel/r2d2/struct.Builder.html
+      POSTGRES_IDLE_TIMEOUT: '60'
+      POSTGRES_MAX_LIFETIME: '120'
+      POSTGRES_CONNECTION_TIMEOUT: '5'
+      POSTGRES_MAX_CONNECTIONS: '3'
+      FOG_PUBKEY_EXPIRY_WINDOW: '10'
+
+
+  externalConfigMaps:
+    # sentry is optional
+    sentry:
+      name: sentry
+      ### required keys:
+      #   fog-ledger-sentry-dsn
+    mobilecoind:
+      name: fog-mobilecoind
+      ### required keys:
+      #   MC_PEER
+      #   MC_TX_SOURCE_URL
+      #   MC_QUORUM_SET
+      ### optional keys:
+      #   MC_LEDGER_DB_URL
+      #   MC_WATCHER_DB_URL
```

Comment:
is this actually needed? wondering because we removed mobilecoind from fog-ingest

Reply:
it looks like it was brought in here https://github.com/mobilecoinfoundation/mobilecoin/pull/3707, which was before the mobilecoind stuff.

@jgreat do you think this is needed anymore, would probably clean up in 5.2.1 if so